test/controllers/registrations_controller_test.rb

```diff
@@ -151,7 +151,7 @@ def teardown
     assert_no_difference 'User.count' do
       post :create, params: { api_user: { password: p1, password_confirmation: p1, email: existing_user.email, login: 'test', name: 'Test' } }
       assert_response 401
-      assert_equal 'Please check your email. If an account with that email doesn’t exist, you should have received a confirmation email. If you don’t receive a confirmation e-mail, try to reset your password or get in touch with our support.', response.parsed_body.dig("errors", 0, "message")
+      assert_equal 'Please check your email. If an account with that email doesn’t exist, you should have received a confirmation email. If you don’t receive a confirmation e-mail, try to reset your password or get in touch with our support.', JSON.parse(response.parsed_body).dig("errors", 0, "message")
```

Comment:
Dumb question: Why did you have to change this test to use JSON.parse?

Reply:
This is the correct behavior; I do not know how this one worked before. :smile: